admin/app/components/autonomous-courses/details.js

```diff
@@ -0,0 +1,44 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+export default class Details extends Component {
+  @tracked isLinkCopied = false;
+
+  constructor() {
+    super(...arguments);
+  }
+  get formattedCreatedAt() {
+    const createdAtDate = new Date(this.args.autonomousCourseDetails.createdAt);
+
+    const day = createdAtDate.getDate().toString().padStart(2, '0');
+    const month = (createdAtDate.getMonth() + 1).toString().padStart(2, '0');
+    const year = createdAtDate.getFullYear();
+
+    return `${day}/${month}/${year}`;
+  }
+
+  get campaignLink() {
+    return `https://app.pix.fr/campagnes/${this.args.autonomousCourseDetails.code}`;
```

Comment:
Ajouter la variable d'env pour les environement.
return `https://${host}/campagnes/${this.args.autonomousCourseDetails.code}`;